Assets/Scripts/Core/GlobalEvent.cs

```diff
@@ -0,0 +1,5 @@
+
+public static class GlobalEvent
+{
+
+}
```

Comment:
Used this pattern in a previous project, it worked really well.

Eventually going to add global events, so that scripts can subscribe, unsubscribe like so:

```
void OnEnable() {
  GlobalEvent.OnPlayerDeath += OnPlayerDeath;
}

void OnDisable() {
  GlobalEvent.OnPlayerDeath -= OnPlayerDeath;
}

void OnPlayerDeath() { ... }
 ```

Simple.